view-app/src/record.js

```diff
@@ -29,7 +28,10 @@ function onstopMediaRecorder() {
   audio.src = audioURL;
   var formData = new FormData();
   formData.append('recordingTemp', blob);
-  postFormData(`http://127.0.0.1:5000/music`, formData);
+  postFormData(`http://127.0.0.1:5000/recorded`, formData).then(() => {
```

Comment:
Explanation: need to wait until the post is completed (and this can last some time, as it sends a file), and only then ask to fetch notes and chords.

Reply:
Why change in endpoint? /music -> /recorded

Reply:
Bc if we used the old one, that would mean uploading the recorded file every time we want to transpose to an easier key or transpose back

Reply:
Ok

Reply:
This needs also change in backend code

Reply:
In fact we could do it the other way around and use post and get requests, where get would mean "use old file"

Reply:
What do you think?